inc/Engine/Admin/PerformanceMonitoring/AJAX/Controller.php

```diff
@@ -303,6 +313,15 @@ public function reset_page() {
 
 		$this->manager->add_url_to_the_queue( $row->url, true ); // @phpstan-ignore-line
 
+		/**
+		 * Fires when a performance monitoring job is reset/retested.
+		 *
+		 * @since 3.20
+		 *
+		 * @param int    $id The database row ID of the reset job.
+		 */
+		do_action( 'rocket_pm_job_added', $id );
```

Comment:
```suggestion
		do_action( 'rocket_pm_job_retest', $id );
```
Should this be retest?